Assets/Scripts/PlayerManager.cs

```diff
@@ -0,0 +1,20 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class PlayerManager : Singleton<PlayerManager> {
+
+    public PlayerController tankController { get; private set; }
```

Comment:
public property should be PlayerController
private variables should be playerController